tools/check_jsonnet

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+success=1
+JSONNET_FILES=$(git ls-files data/ | grep '.json\?net$' | xargs echo)
+
+if test -n "$JSONNET_FILES"; then
+    which jsonnet >/dev/null 2>&1 || (echo "Command 'jsonnet' not found, can not execute Jsonnet syntax checks" && exit 1) || success=0;
```

Comment:
comparing with yamllint, you don't do:
export PERL5LIB=${PERL5LIB_}
I don't know if that could be related with the PATH problem you have, but yamllint is installed in the same way so both executable should behave similar...